README.md

```diff
@@ -20,26 +20,18 @@ The backend is written using a combination of NodeJS, Express, and MongoDB (the
 ## Setup
 
 Dependencies you'll need:
-* Node 12+
-* NPM 6+
-* MongoDB
+
+- Node 12+
+- NPM 6+
+- MongoDB
 
 First, install all Node dependencies:
 
 ```
 npm install
 ```
 
-Next, if you are in a development environment, you can start a "watching" service that will continually
-rebuild the frontend whenever a change is detected. Note that this service is slightly buggy and may not 
-load some pages correctly, particularly with a dynamic URL component. It is recommended that you open
-a separate terminal to run the watcher:
-
-```
-npm run watch
-```
-
-However, if you are in a production environment, you can perform a one-time build of the frontend.
+If you are in a production environment, you can perform a one-time build of the frontend.
 This will not monitor the frontend directory for changes:
```

Comment:
Should this be changed since it does monitor the frontend directory for changes?